precode.sh

```diff
@@ -1,43 +1,65 @@
 #!/bin/bash
 
+
 # создаём каталог task с вложенными директориями
 # task
 #   dir1
 #   dir2
 #   dir3
 #       dir4
+mkdir ./task
```

Comment:
Эту команду можно написать в одну строчку
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```